pkg/utils/resources/resources.go

```diff
@@ -59,6 +59,23 @@ func Merge(resources ...v1.ResourceList) v1.ResourceList {
 	return result
 }
 
+// MergeInto sums the resources from src into dest, modifying dest
+func MergeInto(dest v1.ResourceList, src v1.ResourceList) v1.ResourceList {
```

Comment:
Can we make a comment that this is more performant than the alternative? I can see someone looking at this later and wondering why we have both this and `Merge`

Reply:
Yes, added a comment.